modules/test/playwright/pages/dynamic-data-mapping-form-web/FormBuilderSidePanelPage.ts

```diff
@@ -71,7 +75,7 @@ export class FormBuilderSidePanelPage {
 
 		await this.paragraphFieldTextarea.press('End');
 
-		await this.page.waitForLoadState('networkidle');
+		await this.page.waitForTimeout(1000);
```

Comment:
Can you try replacing this with
```
		await page.waitForResponse(
			'http://localhost:8080/o/dynamic-data-mapping-form-context-provider/'
		);

```

Reply:
hey @rebsilva, if the above works regardless of debounce delay, we could make a utility function like
```
async function waitForDDMEvaluate {
      await page.waitForResponse(
          `${apiHelpers.baseUrl}o/dynamic-data-mapping-form-context-provider/`
      );
}
```
and reuse it everywhere is needed :heart_eyes: